### DISCUSSION TITLE
js13k - update to the current year?

### DISCUSSION THREAD

**Discussion by rockyFierro** [chinchang/web-maker#572] [Ideas]
I'm here because of the gamejam js13k, i only fgound out about webmaker through the jam and I was wondering if the 'mode' could be updated or perhaps even better, a gamejam mode where we could create our own rules and have updates scroll at the bottom or just hover... idk. I just know this is great for rapid prototyping and would easily be the funnest thing for game jams.

**Comment by chinchang** [comment]
The countdown is now updated to reflect the correct number of days. This is already live in the web app and extension will follow.
![CleanShot 2024-08-20 at 14 15 04@2x](https://github.com/user-attachments/assets/6d08e6ca-69e3-4321-b389-ffcba80a5119)

I am curious to understand more about this gamejam mode you talked about! Could you elaborate more on what you have in mind? Or may be point me to something similar, if there is.
